samples/cpp/text_generation/benchmark_genai.cpp

```diff
@@ -30,7 +32,22 @@ int main(int argc, char* argv[]) try {
         return EXIT_SUCCESS;
     }
 
-    std::string prompt = result["prompt"].as<std::string>();
+    std::string prompt;
+    if (result.count("prompt") && result.count("prompt_file")) {
+        std::cout << "Prompt and prompt file should not exist together!" << std::endl;
+        return EXIT_FAILURE;
```

Comment:
If `prompt_file` is given, it's required to explicitly set `prompt` to empty make it work because of the default value. You should set `prompt` to empty by default. Same for VLM.

You already did something similar for `samples/python/text_generation/benchmark_genai.py`, but the problem with `benchmark_genai.py`'s approach is that it's no possible to distinguish the default value and explicit miss use with `The Sky is blue because`. Set the default to None for Python

Reply:
@Wovchena  If the user enters both prompt file and prompt, there is no error, and prompt uses the value of prompt file? And then the prompt which user entered is set to empty?

Reply:
If `prompt` and `prompt_file` are given, it should be an error. My point was about the default values

Reply:
Updated readme.md